src/ReportManager/SchemaFormContext/SchemaForm/fields/fieldDetailsFactory/index.js

```diff
@@ -0,0 +1,12 @@
+
+export const textFieldDetailsFactory = ({ description, title, deprecated, default: defaultValue }, { inputType, placeholder, isRequired }, formValue, formError) => ({
```

Comment:
nit: destructuring here makes this line longer and doesn't provide information on what the arguments of the function are. A developer would have to infer which each of those args is. Why not to call them by their name `(jsonSchema, uiSchema, value, error)`? We can then read them from their objects.

Reply:
Done